tests/reports.test.js

```diff
@@ -0,0 +1,345 @@
+                response.body.should.have.property("message").eq("No se encontro registro(s)");
+                done();
+            });
+        });
+    
+        it("GET /Reports DateCreation Bad Request", (done) =>{
+            chai.request(server).get("/reports/dateCreation/Martha")
+            .auth(accessToken, { type: 'bearer' })
+            .end( (error, response) => {
+                response.should.have.status(400);
+                response.body.should.have.property("message");
+                response.body.should.have.property("message").eq("La fecha debe tener el formato YYYY-MM-DD");
+                done();
+            });
+        });
+    });
+});
+
+describe("GET Tests Reports",() =>{
+    before((done) =>{
+        const login = {
+            username: "Karla",
+	        password: "Marst1245#"
+        };
+        chai.request(server).post("/login").send(login)
+        .end( (error, response) => {
+            accessToken = response.body.token;
+            done();
+        });
+    });
+
+    it("GET /Reports", (done) =>{
+        chai.request(server).get("/reports")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(200);
+            response.body.should.be.a("array");
+            response.body.length.should.be.eq(2);
+            done();
+        });
+    });
+});
+
+describe("GET Tests Report",() =>{
+    before((done) =>{
+        const login = {
+            username: "Karla",
+	        password: "Marst1245#"
+        };
+        chai.request(server).post("/login").send(login)
+        .end( (error, response) => {
+            accessToken = response.body.token;
+            done();
+        });
+    });
+
+    it("GET /Reports/ID", (done) =>{
+        chai.request(server).get("/reports/6169cc2ca32bd265c102ff31")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(200);
+            response.body.should.be.a("object");
+            response.body.should.have.property("_id");
+            response.body.should.have.property("reason");
+            response.body.should.have.property("context");
+            response.body.should.have.property("idAccount");
+            response.body.should.have.property("dateCreation");
+            response.body.should.have.property("accountReported");
+            done();
+        });
+    });
+
+    it("GET /Reports/ID Not Found", (done) =>{
+        chai.request(server).get("/reports/616b0efeba862c9a697da9dc")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(404);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("No se encontro registro(s)");
+            done();
+        });
+    });
+
+    it("GET /Reports/ID Bad Request", (done) =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
    it("GET /Reports/ID Bad Request", (done) => {
```